blockbase/theme.json

```diff
@@ -203,6 +203,9 @@
 				"horizontal": "30px",
 				"vertical": "30px"
 			},
+			"padding": {
```

Comment:
I hesitate to add a custom value for this to blockbase for this. Would it be possible to do this just with block styles rather than a custom variable + CSS?  Perhaps just adding a top padding to the ["core/post-title" block](https://github.com/Automattic/themes/blob/trunk/blockbase/theme.json#L386) would have the intended effect?

Reply:
Yeah, applying a top padding to the 'core/post-title' block should have the same effect. I nearly went a similar route (although still with CSS..), but I added a reuseable variable as the same additional padding needed to be added to the posts in a query block. We could just do the same for these as well though - add the padding to the block in the `theme.json` (I'm guessing so anyway, without looking into it yet!)